### ISSUE TITLE
Hi is this a fully developed game engine?

### ISSUE THREAD

**Issue by Joe23232** [ElhamAryanpur/BlueEngine#11]
Hi is this a fully developed game engine?

**Comment by ElhamAryanpur** [comment]
Hello!

TL;DR: Nope, and the game development branch of Blue Engine isn't either

Long story: The notion of fully developed engine is a bit subjective. I hope these Q.A. below can help understand the state of Blue Engine:

1) Fully developed in sense of foundations? Yes it is.
2) Fully developed in sense of customization? mostly yes
3) Fully developed in sense of features? not yet, a lot more work needs to get done
4) Is this a game engine? No. BlueEngine is a graphics engine, which is more general than a game engine. Although it can be used as game engine too! The parts are there, you just need to glue them yourself. There is however a planned branch that'll be a game engine with Blue Engine as the renderer, so maybe keep an eye on that (Soon).
5) UI? soon
6) Performance? it's good enough for everyone, I can't claim it being the most performant but you shouldn't face issues in a long time.
7) Stable API? Not until 1.0 release
8) Scripting language? Nope, Rust only.
9) Portable? Yes, it can work on Android, IOS, Windows, Linux, MacOS, and web (it has issues on compiling but soon will be resolved).
10) How long have the development been? Around 10 months so far
11) Visual editor? Planned, but not yet started
12) Where to keep an eye on the progress? We have a discord community, most of the talks and progress reports are done there, so maybe [join in](https://discord.gg/s7xsj9q). We'll be happy to have you there :)

**Comment by Joe23232** [comment]
Thanks mate =D

**Comment by ElhamAryanpur** [comment]
no worries :)
